psydac/linalg/tests/test_block.py

```diff
@@ -989,6 +989,34 @@ def test_block_linear_operator_parallel_dot( dtype, n1, n2, p1, p2, P1, P2 ):
     # Check data in 1D array
     assert np.allclose( Y.blocks[0].toarray(), y1.toarray(), rtol=1e-14, atol=1e-14 )
     assert np.allclose( Y.blocks[1].toarray(), y2.toarray(), rtol=1e-14, atol=1e-14 )
+
+    # Test copy with an out 
+    K = BlockLinearOperator( W, W )
+    L.copy(out=K)
```

Comment:
I am not convinced that this is the test we want. The `K` object is created completely empty, with no blocks in its dictionary. Hence when we copy `L` into `K` we are creating all new `StencilMatrix` blocks instead of writing into preexisting blocks.